### PR TITLE
Fix antora versioning

### PR DESCRIPTION
**Pull request by chibenwa**
Screenshot from 2024-07-02 09-41-30

### CODE REVIEW

File: doc-sites/antora-playbook.yml [hunk 1]
```diff
@@ -7,10 +7,30 @@ site:
   start_page: james-site::homepage.adoc
 content:
   sources:
-    - url: https://github.com/apache/james-project.git
+    - url: https://github.com/chibenwa/james-project.git
```
Comment (Arsnael) [inline doc-sites/antora-playbook.yml]: should be apache repo no? Idem for other items?

Comment (chibenwa) [reply doc-sites/antora-playbook.yml]: Correct. I WIP this then.

Comment (Arsnael) [reply doc-sites/antora-playbook.yml]: The versioning fix PRs have been merged on all concerned branches in Apache james repo, so I guess we can switch to apache repo here, and it should build us the proper antora doc we want correct? :)

Comment (chibenwa) [inline doc-sites/antora-playbook.yml]:
```suggestion
    - url: https://github.com/apache/james-project.git
```

File: doc-sites/antora-playbook.yml [hunk 2]
```diff
@@ -7,10 +7,30 @@ site:
...
       branches: master
       start_path: docs
       edit_url: "{web_url}/blob/{refname}/{path}"
+    - url: https://github.com/chibenwa/james-project.git
```
Comment (chibenwa) [inline doc-sites/antora-playbook.yml]:
```suggestion
    - url: https://github.com/apache/james-project.git
```

File: doc-sites/antora-playbook.yml [hunk 3]
```diff
@@ -7,10 +7,30 @@ site:
...
+      branches: 3.8.x
+      start_path: docs
+      edit_url: "{web_url}/blob/{refname}/{path}"
+    - url: https://github.com/chibenwa/james-project.git
```
Comment (chibenwa) [inline doc-sites/antora-playbook.yml]:
```suggestion
    - url: https://github.com/apache/james-project.git
```

File: doc-sites/antora-playbook.yml [hunk 4]
```diff
@@ -7,10 +7,30 @@ site:
...
+      branches: 3.7.x
+      start_path: docs
+      edit_url: "{web_url}/blob/{refname}/{path}"
+    - url: https://github.com/chibenwa/james-project.git
```
Comment (chibenwa) [inline doc-sites/antora-playbook.yml]:
```suggestion
    - url: https://github.com/apache/james-project.git
```

File: doc-sites/antora-playbook.yml [hunk 5]
```diff
@@ -7,10 +7,30 @@ site:
...
+      branches: 3.8.x
+      start_path: server/apps/distributed-app/docs
+      edit_url: "{web_url}/blob/{refname}/{path}"
+    - url: https://github.com/chibenwa/james-project.git
```
Comment (chibenwa) [inline doc-sites/antora-playbook.yml]:
```suggestion
    - url: https://github.com/apache/james-project.git
```

File: doc-sites/antora-playbook.yml [hunk 6]
```diff
@@ -7,10 +7,30 @@ site:
...
+      branches: 3.7.x
+      start_path: server/apps/distributed-app/docs
+      edit_url: "{web_url}/blob/{refname}/{path}"
+    - url: https://github.com/chibenwa/james-project.git
```
Comment (chibenwa) [inline doc-sites/antora-playbook.yml]:
```suggestion
    - url: https://github.com/apache/james-project.git
```

**Review by Arsnael — APPROVED** [review]

**Review by vttranlina — APPROVED** [review]

**Pull request merged**